ios general/ios general/CreateEmployeeViewController.m

```diff
@@ -0,0 +1,60 @@
+#import "CreateEmployeeViewController.h"
+
+@interface CreateEmployeeViewController ()
+
+@end
+
+@implementation CreateEmployeeViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.title = @"Add new Employee";
+    
+    [self.firstNameField addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    [self.lastNameField  addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    [self.salaryField    addTarget:self action:@selector(textFieldDidChange:) forControlEvents:UIControlEventEditingChanged];
+    
+    UIBarButtonItem *saveButton = [[UIBarButtonItem alloc] initWithBarButtonSystemItem:UIBarButtonSystemItemSave target:self action:@selector(saveNewObject:)];
+    self.navigationItem.rightBarButtonItem = saveButton;
+}
+
+- (void)didReceiveMemoryWarning
+{
+    [super didReceiveMemoryWarning];
+}
+
+-(BOOL)textFieldDidChange:(UITextField *)textField
+{
+    if ([textField.text isEqualToString:@""])
```

Comment:
This will work, but is a bit incorrect, since textField.text may be nil, so isEqualToString:@"" won't work. To check for an empty string in Objective-C, use: `string.length > 0` - it will work correctly for both empty strings and nils.